loadtests/simulation.py

```diff
@@ -0,0 +1,318 @@
+import json
+import os
+import random
+import uuid
+
+from konfig import Config
+from loads.case import TestCase
+from requests.auth import HTTPBasicAuth
+
+
+ACTIONS_FREQUENCIES = [
```

Comment:
I am wondering if we could write it in a way it does try each of them at least once so that we can be confident about the stability of the loadtest.

I'd like to run it once as a smoketest here while making sure each call was tried at least once.